build.py

```diff
@@ -130,6 +130,8 @@ class HelpFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescript
 
     parser.add_argument("--use_dml", action="store_true", help="Whether to use DML. Default is to not use DML.")
 
+    parser.add_argument("--disable_guidance", action="store_true", help="Whether to add guidance support. Default is True.")
```

Comment:
Maybe just `use_guidance` and make the default to `true`. There's no `disable_sth` argument until now.